modules/API/pages/built-in-endpoints.adoc

```diff
@@ -2579,7 +2750,7 @@ curl -X GET "localhost:9000/abortquery/poc_graph?requestid=16842763.RESTPP_1_1.1
 
 | `url`
 | No
-| The endpoint whose running queries to abort. You must specify the base of the endpoint's URL, but then use a wildcard to allow for different parameters. For example, to abort all running queries for the endpoint `/sleepgpe`, use `url=/sleepgpe.*`
+| The endpoint on which to abort running queries. You must specify the base of the endpoint's URL, but then use a wildcard to allow for different parameters. For example, to abort all running queries for the endpoint `/sleepgpe`, use `url=/sleepgpe.*`
```

Comment:
I think this rephrasing is not correct. 

The original meaning is that if you have a query (NOT necessarily a GSQL query - this is confusing and we should find a better way of saying this. Query here basically refers to a request) that is running on RESTPP, you can abort it by specifying the URL of the request. So the query here belongs to the endpoint - and that possessive relationship is important.

Reply:
I thought for quite a while but couldn't come up with anything yet to improve the original, so reverted.